fuel/transformers/image.py

```diff
@@ -0,0 +1,263 @@
+from __future__ import division
+from io import BytesIO
+import math
+
+import numpy
+from PIL import Image
+from six import PY3
+
+from ._image import window_batch_bchw
+from . import ExpectsAxisLabels, SourcewiseTransformer
+from .. import config
+
+
+class ImagesFromBytes(SourcewiseTransformer):
+    """Load from a stream of image-formatted data represented as bytes.
```

Comment:
It's not very clear what is 
> represented as bytes
Do you mean that it can read any kind of image file?